native/packages/react-native-bpk-component-text-input/src/BpkTextInput.js

```diff
@@ -79,6 +79,7 @@ const styles = StyleSheet.create({
     // TODO Replace with fontFamilyLg token once available
     fontFamily: Platform.OS === 'android' ? 'sans-serif-medium' : 'System',
     lineHeight: textLgLineHeight,
+    height: '100%',
```

Comment:
I'd expect to see a snapshot update for this.

Reply:
Soon Danger will do this!

Reply:
But Travis already does!

Reply:
I feel like instead of doing this, we should have a touch-listener on the entire component which activated the TextInput.
Thoughts?